dataactvalidator/config/sqlrules/a34_appropriations.sql

```diff
@@ -0,0 +1,12 @@
+SELECT
+    approp.row_number,
+    approp.budget_authority_unobligat_fyb,
+    sf.amount as sf_133_amount
```

Comment:
The rule text has the line # in there, so I think that would be enough to track down the error. In previous rules, we haven't included the line number in the column alias, so if we decide to go that route, those would need to be updated as well.